schematics/src/ng-add/index.ts

```diff
@@ -54,15 +56,24 @@ export function addImportsToModuleDeclaration(options: SchemaOptions, imports: s
   };
 }
 
+function addModuleToPackageJson() {
+  return (host: Tree, context: SchematicContext) => {
+    addPackageToPackageJson(host, "dependencies", "@ngneat/overview", "1.0.0");
```

Comment:
I have a problem with hard coded version "1.0.0" here. @NetanelBasal Do you have an idea how we can get the latest version to install?

Reply:
Use `^1.0.0`

Reply:
Or use https://www.npmjs.com/package/latest-version

Reply:
Ok, currently I have used ^1.0.0